app/helpers/application_helper.rb

```diff
@@ -14,4 +14,8 @@ def i18n_pluralize(word)
   def i18n_error_count(count)
     I18n.locale == :ja ? "#{count}件の#{t('views.common.error')}" : pluralize(count, t('views.common.error'))
   end
+
+  def show_user(instance)
```

Comment:
instanceだと何のインスタンスを想定しているのかわかりにくいです。
また、userを呼び出せないインスタンスを渡すとエラーが発生します。

そのために、ここはUserオブジェクト（Userのインスタンス）を渡す設計にした方がいいです。

```suggestion
  def show_user(user)
```

また、メソッド名についてですが、「show user = ユーザーを表示」だとちょっと抽象的過ぎる気がしますね。
https://bootcamp.fjord.jp/articles/78

このメソッド自体が何かを画面に表示しているわけではない、という指摘もできそうです。
というわけでメソッド名も見直してみてください〜。